### PR TITLE
RSS url needed the relative path removed to stage it correctly

### PR DESCRIPTION
**Pull request by timsaucer**
This is a one line change to adjust the RSS url. It is currently being put into a blog subdirectory instead of the output directory, causing it to need an extra `/blog` in the URL on the main site.

### CODE REVIEW

**Comment by timsaucer** [comment]
There is still a problem with the new site serving up when I attempt to remove the old site content. I have opened a ticket with the infra team: https://issues.apache.org/jira/browse/INFRA-26350

**Comment by timsaucer** [comment]
@alamb I believe this is ready to go. I resolved the issue with the infra team. We had to make an adjustment to the .asf.yaml file within the `asf-site` branch. I had to make some updates to the paths so that they process correctly, but I did test this the staging branch and I've also pushed the update to the site branch.

File: README.md [hunk 1]
```diff
@@ -36,12 +46,14 @@ Then within the directory that contains `datafusion-site` you can build and test
 the site using:
 
 ```shell
-docker run --rm -it -p8000:8000 -v $PWD:/site df-site-build:latest
+docker run -it --rm -p8000:8000 -v $PWD:/site --entrypoint /bin/bash df-site-build:latest
```
Comment (alamb) [inline README.md:49]: I tried this out and it works great

**Review by alamb — APPROVED** [review]
Looks good to me -- thank you @timsaucer

**Pull request merged**